src/universal/components/DashboardWrapper/DashboardWrapper.js

```diff
@@ -44,6 +44,7 @@ const teamRoot = () =>
 const newTeam = () =>
   import(/* webpackChunkName: 'NewTeamRoot' */ 'universal/modules/newTeam/containers/NewTeamForm/NewTeamRoot')
 
+// TODO remove dispatch when we completely remove redux (welcome, filters, etc. still need it)
```

Comment:
We should probably create issues for "refactor welcome experience to remove redux" and "refactor Kanban filters to use relay"...